lib/wordmove/movefile.rb

```diff
@@ -26,6 +26,17 @@ def fetch(verbose = true)
       YAML.safe_load(ERB.new(File.read(found)).result, [], [], true).deep_symbolize_keys!
     end
 
+    def dotenv(cli_options = {})
+      env = cli_options['environment'] || cli_options[:environment]
+      env_files = Dir[File.join(start_dir, ".env{.#{env},}")]
+
+      found_env = env_files.first
+      logger.task("Using .env file: #{found_env}") if found_env && found_env != ENV['dotenv']
+      ENV['dotenv'] = found_env
```

Comment:
Would you mind to explain to me how you think `ENV['dotenv']` could be sued/useful?

At first glance I'd have wrote

```ruby
found_env = env_files.first

return false unless found_env.present?

logger.task("Using .env file: #{found_env}")
Dotenv.load(found_env) # I hope .load will return true/false thus having a consistent return value
```

But I'm afraid I'm missing something from your reasoning.

Reply:
My reasoning was to serve a dual purpose:
* A quasi-immutable value to ensure stdout doesn't print the "Using .env file" statement more than once (something I ran into a few times when developing the feature).
* Saving the path for the dotenv could be useful to some users trying to extend the functionality of the movefile like so:

```yaml
hooks:
  push:
    before:
      local:
        - 'yarn run build --env="<%= ENV['dotenv'] %>"'
```

...but after writing it out, I realize the first point is counter-productive, and the second is pointless without actually documenting it. Plus, it's kinda enabling some potentially bad habits. Removed!